erlang/README.md

```diff
@@ -60,18 +62,46 @@ Use a serial console program, such as `minicom`, to attach to the device over US
 
 ### Running on the STM32 platform
 
-TODO
+To run this application on the STM32 platform, you must flash the application to the device attached to your computer via USB.  You may then optionally monitor the program via a serial console program to view any data output to the console.
+
+#### Flashing onto an STM32 Device
+
+To flash this application to your ESP32 device, issue the `stm32_flash` target.  Use the `--port` option to specify the port to which your device is connected:
+
+    shell$ rebar3 atomvm stm32_flash --port /dev/ttyUSB0
+
+#### Monitoring an STM32 Device
+
+Use a serial console program, such as `minicom`, to attach to the device over USB:
+
+    shell$ minicom -D /dev/ttyUSB0
 
 ### Running on the Raspberry Pico platform
 
-TODO
+To run this application on the Raspberry Pico platform, you must flash the application to the device attached to your computer via USB.  You may then optionally monitor the program via a serial console program to view any data output to the console.
+
+#### Flashing onto a Raspberry Pico Device
+
+To flash this application to your ESP32 device, issue the `esp32_flash` target.  Use the `--port` option to specify the port to which your device is connected:
+
+    shell$ rebar3 atomvm pico_flash
+
+Note that your Raspberry Pico must be in bootloader mode (press and hold the BOOTSEL button while powering on the device), and the device must be mounted via USB.  After the Raspberry Pico has been flashed, it should automatically unmount the the USB volume and restart in
```

Comment:
This is not necessary except for custom builds configured with the cmake option `-DAVM_WAIT_FOR_USB_CONNECT=on`. For default builds the device will automatically be reset into `BOOTSEL` mode. @pguyot and I are discussing the best way to fix that problem so that it is never an issue.

Reply:
Maybe I have the terminology wrong.  The point I was trying to make is that the Pico must be mounted as a volume in order to run the pico_flash task.  I have re-worded it to hopefully make it more clear.

Reply:
Those steps are only required to flash the VM and libraries. The `pico_flash` task will automatically put the device into `BOOTSEL` - at least with a default build of the VM. 

Users that have built a custom VM with the `AVM_WAIT_FOR_USB_CONNECT=on` are the only ones that need to unplug and manually put the device into `BOOTSEL` mode.

Reply:
I don't believe that is true on MacOS.  Or, at least, I have not been able to get it to work that way on that platform.

Reply:
Removed the section.  Let users sort it out.